tests/openvino/native/test_model_transformer.py

```diff
@@ -162,3 +165,42 @@ def test_fq_insertion_weights(target_layers, ref_fq_names):
     assert len(fq_nodes) == len(ref_fq_names)
     for fq_name in fq_nodes:
         assert fq_name in ref_fq_names
+
+
+MODELS_WITH_PARAMETERS = [
+    {
+        'model': ConvModel().ov_model,
+        'layers': ['Conv'],
+        'values': [np.full((3,), 2)],
+        'refs': [2.0],
+    },
+    {
+        'model': FPModel(precision='FP16').ov_model,
+        'layers': ['MatMul'],
+        'values': [np.full((3,), 2)],
+        'refs': [2.0],
+    }
+]
+
+
+@pytest.mark.parametrize('model_with_parameters', MODELS_WITH_PARAMETERS)
+def test_bias_correction(model_with_parameters):
```

Comment:
We need the same test for ONNX

Reply:
If you look into the code, you will find that this test already exists for the ONNX backend.

Reply:
I see